jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -824,7 +823,7 @@ public PaymentJson apply(final Payment payment) {
         return Response.status(Response.Status.OK).entity(result).build();
     }
 
-    @Timed
+    @TimedResource
```

Comment:
You probably want a `@MetricTag` here, as this will trigger an AUTH, PURCHASE or CREDIT.